mlflow/store/artifact/ftp_artifact_repo.py

```diff
@@ -112,6 +112,7 @@ def list_artifacts(self, path=None):
                 return []
             artifact_files = ftp.nlst(list_dir)
             artifact_files = list(filter(lambda x: x != "." and x != "..", artifact_files))
+            artifact_files = list(map(lambda x: os.path.basename(x), artifact_files))
```

Comment:
Can you insert a comment on why we need to call `os.path.basename(x)`  above this line?

Reply:
ok

Reply:
I will insert a comment as follows:

> This is to make sure artifact_files is a list of the file names, because ftp.nlst may return them with an absolute path.

What do you think?

Reply:
How about:
```
Make sure artifact_files is a list of files names because ftp.nlst may return absolute paths.
```
?